### PR TITLE
Bumping version after publication and adding contribution guidelines

### PR DESCRIPTION
**Pull request by elazzabi**
After the publication, bumped the version and added a contribution guidelines

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Contributing
+
+## Releasing / Publishing
+
+### New release
+Publishing a new version can be done using the following checklist:
+
+1. Set the version (via `npm version minor` or `npm version major` or `npm version patch`)
+2. For most regular releases, this will be `npm version minor`.
+3. By default, npm publish scaped modules as private. To make it public, make sure to add the `access` flag. The whole command is: `npm run publish-please --access public`
```
Comment (nickdaugherty) [inline CONTRIBUTING.md]: > npm publish scaped modules

Small typo here

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+4. Push the tag to GitHub (`git push --tags`)
+5. Edit the release on Github to include a description of the changes and publish
+6. Bump the version to the next preminor: `npm --no-git-tag-version version preminor`
```
Comment (nickdaugherty) [inline CONTRIBUTING.md]: Can you add an explanation of what this does?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+7. Make a new PR with the changes (`git add -u` + `git commit` + `git push origin master`).
```
Comment (nickdaugherty) [inline CONTRIBUTING.md]: It's unclear to me what this step is for (and it's pushing to `master` rather than creating a PR).

**Comment by elazzabi** [comment]
Thanks @nickdaugherty , made the changes here d61fb4857b570f063bf95b44d8520df8190a4d84 do they look good?

**Pull request merged**